tasks/main.yml

```diff
@@ -10,9 +10,11 @@
   file:
     path: "{{ homebrew_prefix }}"
     mode: 0775
+    owner: root
     group: admin
     state: directory
   become: yes
+  become_user: root
```

Comment:
`become` usually defaults to `root`, though—or if globally configured otherwise, that different user... I'd rather either leave this as-is, or switch it to a configurable user, defaulted to `root`.